lib/src/screens/ur/animated_ur_page.dart

```diff
@@ -160,6 +179,24 @@ class _URQRState extends State<URQR> {
     super.dispose();
   }
 
+  late String selected = (widget.urqr.isEmpty) ? "unknown" : widget.urqr.keys.first;
+  int selectedInt = 0;
+
+  List<String> get frames {
+    return widget.urqr[selected]?.split("\n") ?? [];
+  }
+
+  late String nextLabel = widget.urqr.keys.toList()[(selectedInt + 1) % widget.urqr.length] ;
+
+  void next() {
+    final keys = widget.urqr.keys.toList();
+    selectedInt++;
+    setState(() {
+      nextLabel = keys[(selectedInt) % keys.length];
+      selected = keys[(selectedInt) % keys.length];
```

Comment:
I think this is not what you intended right?

Reply:
`selectedInt++;` should be in setState - other than that it is good